cerbero/commands/package.py

```diff
@@ -41,6 +42,9 @@ def __init__(self):
             ArgparseArgument('-t', '--tarball', action='store_true',
                 default=False,
                 help=_('Creates a tarball instead of a native package')),
+            ArgparseArgument('-z', '--zip', action='store_true',
```

Comment:
Previously there were only 2 choices, either normal package or tarball, so the --tarball switch was enough. If you add another switch for zip, you could endup with --tarball --zip and generate only a tarball. I would use a "choice" option:
 ArgparseArgument('-t', '--type', type=str, choices=['native', 'tarball', 'zip'], default='native'))
These cahnge is not backwards comatible and we will have to fix the buildbots that uses it too.